src/odemis/driver/pigcs.py

```diff
@@ -805,9 +804,13 @@ def GetMotionStatus(self, check=True):
         mv_axes = set()
         while bitmap > 0:
             if bitmap & 1:
-                mv_axes.add(i)
+                mv_axes.add(bytes(i))
             i += 1
             bitmap >>= 1
+        # if axes labels are not digits convert the mv_axes to characters.
+        if not all(k.isdigit() for k in self._channels):
+            # Assume axes are sorted alphabetically, i.e. 1 corresponds to x, 2 to y 3 to z.
+            mv_axes = set(sorted(self._channels)[:int(max(mv_axes))])
```

Comment:
I think this is the correct way:
`set([sorted(self._channels)[int(ax) - 1] for ax in mv_axes])`

Reply:
I just proposed something else. I think both would work. The tiny advantage of mine is that if bitmap is a crazy value, it'll not cause this mapping to fail. 
BTW, in your version you can drop the `[]`.